src/m365/planner/commands/plan/plan-get.ts

```diff
@@ -0,0 +1,160 @@
+import { Logger } from '../../../../cli';
+import {
+  CommandOption
+} from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import Utils from '../../../../Utils';
+import GraphCommand from '../../../base/GraphCommand';
+import commands from '../../commands';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  id?: string;
+  title?: string;
+  ownerGroupId?: string;
+  ownerGroupName?: string;
+}
+
+class PlannerPlanGetCommand extends GraphCommand {
+  public get name(): string {
+    return commands.PLAN_GET;
+  }
+
+  public get description(): string {
+    return 'Get a Microsoft Planner plan';
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.id = typeof args.options.id !== 'undefined';
+    telemetryProps.title = typeof args.options.title !== 'undefined';
+    telemetryProps.ownerGroupId = typeof args.options.ownerGroupId !== 'undefined';
+    telemetryProps.ownerGroupName = typeof args.options.ownerGroupName !== 'undefined';
+    return telemetryProps;
+  }
+
+  public defaultProperties(): string[] | undefined {
+    return ['id', 'title', 'createdDateTime', 'owner'];
+  }
+
+  public commandAction(logger: Logger, args: CommandArgs, cb: () => void): void {
+    if (args.options.id) {
+      this
+        .getPlan(args)
+        .then((res: any): void => {
+          logger.log(res);
+          cb();
+        }, (err: any): void => this.handleRejectedODataJsonPromise(err, logger, cb));
+    }
+    else {
+      this
+        .getGroupId(args)
+        .then((groupId: string): Promise<any> => {
+          const requestOptions: any = {
+            url: `${this.resource}/v1.0/groups/${groupId}/planner/plans`,
```

Comment:
Will this still work if the group has more plans than fits in a single response? Shouldn't we use `getAllItems` here?

Reply:
Tested with 2 plans with same name in the same group and apparently it still works.

![image](https://user-images.githubusercontent.com/36161889/126163248-d31c2e48-b48a-4b54-8c7a-937db40e7e4d.png)

Reply:
I will try to change it with `getAllItems`. I thought plans had to have unique names, but guess they don't 😄